backend/src/api.py

```diff
@@ -66,7 +66,18 @@
 
 LOG_DETECTIVE_TOKEN = os.environ.get("LOG_DETECTIVE_TOKEN")
 
-app = FastAPI()
+app = FastAPI(
+    title="Log Detective Website",
+    contact={
+        "name": "Log Detective developers",
+        "url": "https://github.com/fedora-copr/logdetective-website",
+        "email": "copr-devel@lists.fedorahosted.org",
+    },
+    license_info={
+        "name": "Apache 2.0",
```

Comment:
IMO this should be SPDX ID: Apache-2.0
https://spdx.org/licenses/Apache-2.0.html

Reply:
Thank you, the format is now compliant.